cdp_scrapers/instances/portland-static.json

```diff
@@ -0,0 +1,94 @@
+{
+    "persons": {
+        "Ted Wheeler": {
+            "name": "Ted Wheeler",
+            "is_active": true,
+            "router_string": null,
+            "email": "wheelermediarequests@portlandoregon.gov",
+            "phone": "503-823-4120",
+            "website": "https://www.portland.gov/wheeler",
+            "picture_uri": "https://www.portland.gov/sites/default/files/styles/4x5_320w/public/2020-03/mayor-wheeler-headshot-4x5-3-5-2020.jpg",
+            "seat": {
+                "name": "Mayor",
+                "electoral_area": "Citywide",
+                "electoral_type": null,
+                "image_uri": "https://www.portland.gov/sites/default/files/styles/facebook/public/2020-01/city_hall_portland_oregon_2012.jpg",
+                "external_source_id": null,
+                "roles": ["City Budget", "Management and Finance", "Government Relations", "City Attorney", "Police", "Prosper Portland", "Emergency Management", "Equity and Human Rights"]
```

Comment:
Sorry Shak, `seat.roles` are `List[Role]`, defined [here](https://councildataproject.org/cdp-backend/cdp_backend.pipeline.html#cdp_backend.pipeline.ingestion_models.Role). Unfortunately it is not a simple `List[str]`. Sorry! :sweat_smile: If you want to leave these as `null`, that's fine for now.

Reply:
See my comment above about the `roles`.